### PR TITLE
Fixed merge conflict: Add index for TaskSearchController to display all tasks  issue #180

### PR DESCRIPTION
**Pull request by alabamustapha**
Add index for TaskSearchController to display all tasks

issue https://github.com/zurichat/zc_plugin_todo/issues/180
live link: https://todo.zuri.chat/api/tasks

### CODE REVIEW

**Comment by kayleb01** [comment]
What issue are you working on??
your title says nothing about it

**Comment by alabamustapha** [comment]
Add index for TaskSearchController to display all tasks issue #180

File: app/Http/Controllers/Api/TaskSearchController.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        $this->taskService = $taskService;
+    }
+
+    /**
+     * get all tasks.
+     *
+     * @return mixed
+     */
+
```
Comment (kayleb01) [inline app/Http/Controllers/Api/TaskSearchController.php:22]: Your task is to search even though the search has been written already... 
This code you wrote only get all tasks

**Comment by kayleb01** [comment]
The search has  already been written, your work here gets all the tasks and nothing more

File: app/Http/Controllers/Api/TaskSearchController.php (hunk 1 above)
Comment (alabamustapha) [reply app/Http/Controllers/Api/TaskSearchController.php:22]: I only did as the title read. is there any available backend tickets that have not been done?

**Pull request closed without merging**